packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -250,6 +250,9 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
       this.relativeInstalledPackagesPath
     );
 
+    // remove any previous project at this path location
+    shell.rm('-rf', projectPath);
```

Comment:
Do we want to warn user that this actually removes everything vs. merging and overwrite the files?

Reply:
This is the text presented to the user when the project already exists. An overwrite does imply that we will be wiping out the previous logic. However, since this has behaved as a merge for quite some time it might be worth having an additional informative message that this logic has changed to a full overwrite. What do you think?
![Screen Shot 2020-08-11 at 8 57 56 AM](https://user-images.githubusercontent.com/45604118/89913449-f9ce5f00-dbb0-11ea-9c21-cfe3c8058169.png)